crates/sui-framework/sources/TxContext.move

```diff
@@ -130,6 +126,11 @@ module Sui::TxContext {
         ID::new(derive_id(*&self.tx_hash, ids_created - 1))
     }
 
+    #[test_only]
+    public fun advance_epoch(self: &mut TxContext) {
```

Comment:
Might want to rename it to be something more explicit, such as increment_epoch_number, because the actual `advance_epoch` in the governance contract does a lot more.